openage/convert/processor/export/texture_merge.pyx

```diff
@@ -143,4 +144,9 @@ cdef void cmerge_frames(texture, packer_type=PackerType.BINPACK, cache=None) exc
     if isinstance(packer, BestPacker):
         # Only generate these values if no custom packer was used
         # TODO: It might make sense to do it anyway for debugging purposes
-        texture.best_packer_hints = packer.get_mapping_hints(frames)
+        texture.best_packer_hints = packer.get_mapping_hints(blocks)
+
+cdef struct block:
+    unsigned int index
+    unsigned int width
+    unsigned int height
```

Comment:
You should move this to `binpack.pxd`. Then you don't need even need to define `texture_merge.pxd` as you can just import `block` from `binpack.pxd`.